Description: Ignore runstatedir in check-cmdline-options-doc.sh
 autoconf 2.70 will add a --runstatedir option, Debian has backported it.
 Ignore it when checking whether INSTALL is up to date.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Applied-Upstream: http://repo.or.cz/wmaker-crm.git/commit/4f18b3e
Last-Update: 2016-02-09

--- wmaker-0.95.7.orig/Makefile.am
+++ wmaker-0.95.7/Makefile.am
@@ -106,6 +106,7 @@ configure-documentation:
 		--ignore-prg 'enable-shared,enable-static # should be in INSTALL' \
 		--ignore-prg 'disable-option-checking,enable-fast-install # should be in INSTALL' \
 		--ignore-prg 'disable-libtool-lock,with-pic,with-gnu-ld,with-sysroot # for libtool' \
+		--ignore-prg 'runstatedir #new in autoconf 2.70, backported in Debian' \
 		--ignore-prg 'with-x # no use, it would not work without X'
 
 .PHONY: configure-documentation
